apistructs/extention.go

```
@@ -78,6 +78,7 @@ func (spec *Spec) CheckDiceVersion(versionStr string) bool {
 	if err != nil {
 		return true
 	}
+	newSemVersion, _ := semVersion.SetPrerelease("")
```

why you eat the error?